xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -581,6 +580,27 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       m_formatname = "amc-vc1";
       break;
     }
+    case AV_CODEC_ID_AV1:
+    {
+      switch(hints.profile)
+      {
+        case FF_PROFILE_AV1_MAIN:
+          profile = CJNIMediaCodecInfoCodecProfileLevel::AV1ProfileMain8;
+          break;
+        case FF_PROFILE_AV1_HIGH:
+        case FF_PROFILE_AV1_PROFESSIONAL:
+          goto FAIL;
+          break;
+        default:
```

Comment:
Are you sure with the default case here? Shouldn't that go to fail as well? Else profile would be unset?

Reply:
No not quite sure, I tried to see how we do it for other codecs but seems we vary a bit.
I basically stole from h.264 where we support a bunch of profiles and fail for others and the default case is empty.

Reply:
In general I'm not sure how to deal with the profiles: https://developer.android.com/reference/android/media/MediaCodecInfo.CodecProfileLevel#AV1ProfileMain10

Although I'm just using Profile8 atm, I can still decode 10bit files just fine, so I decided to PR it and get some comments.